site/src/docs/components/step-by-step/index.mdx

```diff
@@ -0,0 +1,139 @@
+---
+slug: '/components/step-by-step'
+title: 'StepByStep'
+navTitle: 'StepByStep'
+---
+
+import { StepByStep } from 'hds-react';
+
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview>
+  <StepByStep
+    numberedList
+    title="Numbered step by step component"
+    helpText="Numbered list is suitable for case where the order of the steps is important."
+    steps={[
+      {
+        title: 'Step title',
+        description:
+          "Here you can describe the step in detail. Keep the text compact so the user gets the big picture of the whole process and it's steps easily.",
+        buttons: [
+          {
+            children: 'Example button',
+            href: 'https://hel.fi',
+          },
+        ],
+      },
+      {
+        title: 'Step title',
+        description: 'Step description.',
+        links: [
+          {
+            children: 'Example link',
+            href: 'https://hel.fi',
+          },
+        ],
+      },
+      {
+        title: 'Step title',
+        description: 'Step description.',
+      },
+    ]}
+  />
+</PlaygroundPreview>
+
+### Principles
+
+- The purpose of the StepByStep component is to provide an easy overall picture of a process.
+- **Every step should have a title.**
+- Step description is optional. The description should be compact, preferably one sentence long and two at the maximum. This is because the steps should provide a quick way to understand the whole process and too long texts can hinder that experience.
+- A step can have a button as a call to action, e.g. making a phone call.
+- A step can have a link to provide additional information.
+- The button and the link should relate to the step's contents. We recommend to have only one button or a link per step but sometimes it could be reasonable to have two. Before you add a second button or a link though, think whether the step should be divided in two steps.
```

Comment:
Grammarly suggests to paraphrase this section as following:

"The button and the link should relate to the step's contents. We recommend having only one button or a link per step, but sometimes it could be reasonable to have two. Before adding a second button or a link, consider whether the step should be divided into two steps."

Reply:
Good looking out! [a5af993](https://github.com/City-of-Helsinki/helsinki-design-system/pull/1004/commits/a5af993aceac4a4a15355606bc8743bfc43f7763)